go/test/endtoend/recovery/unshardedrecovery/recovery.go

```diff
@@ -24,6 +24,7 @@ import (
 	"os/exec"
 	"path"
 	"testing"
+	time2 "time"
```

Comment:
any need for renaming the import package

Reply:
Fixed via 013d4047e5